ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -85,6 +85,16 @@ tests:
     test:
     - ref: openshift-qe-ingress-perf
     workflow: rosa-aws-sts
+- as: perfscale-rosa-multiaz-network-perf-3nodes-periodic
+  cron: 0 13 * * 5
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: nightly
+      OPENSHIFT_VERSION: "4.12"
+    test:
+    - ref: openshift-qe-network-perf
+    workflow: rosa-aws-sts
```

Comment:
@jtaleric has a [pr open](https://github.com/openshift/release/pull/41453/files) with almost the exct same configuration. Is there a difference in these configs?

Reply:
Yes, this PR is different and tackles the following configuration in AWS:
1) Rosa cluster in multi-az
2) Self-managed cluster based on the Airflow-DAG
https://github.com/cloud-bulldozer/airflow-kubernetes/blob/master/dags/openshift_nightlies/config/benchmarks/data-plane-v2.json 

Whereas this is [GCP-pr](https://github.com/openshift/release/pull/41453/files)